server/__init__.py

```diff
@@ -5,15 +5,11 @@
 
 app = Flask(__name__)
 
+app.config["SECRET_KEY"] = "68a2c99120b7e9558c126213b6989f1d"
```

Comment:
:thinking: Hm, I think this does not become secret anymore if it is commited. [Flask documentation about SECRET_KEY](https://flask.palletsprojects.com/en/2.0.x/config/#SECRET_KEY) also states that this shouldn't be commited:

> **Do not reveal the secret key when posting questions or committing code.**

 I think this should be loaded from a `.env` file or some kind .conf-like file (be it JSON, INI, TOML..., your choice :) ), which then wouldn't be versioned (and would be added to `.gitignore`).

Reply:
Ohh yeah right....my bad
I'll do the changes